pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -304,12 +304,11 @@ func (service *SecurityPolicyService) populateRulesForIsolationSection(spIsolati
 }
 
 func (service *SecurityPolicyService) generateSectionForNetworkPolicy(networkPolicy *networkingv1.NetworkPolicy, sectionType string) (*v1alpha1.SecurityPolicy, error) {
```

Comment:
Do you think we can add a test case to verify the final NSX security policy resources' ID and Display Name with an input of type NetworkPolicy CR?